app/javascript/modules/listingDetailsAside/ListingDetailsApplicationDate.tsx

```diff
@@ -54,9 +54,9 @@ const getStatusLottery = (listing: RailsListing) => {
 }
 
 const getStatusFcfs = (listing: RailsListing) => {
-  const isClosed = listing.Listing_State === LISTING_STATES.CLOSED
+  const isClosed = !listing.Accepting_Online_Applications
   const datetime = listing.Application_Start_Date_Time
-  const showDateTime = listing.Listing_State === LISTING_STATES.NOT_YET_OPEN
```

Comment:
question: can we remove `Listing_State` from the `RailsListing` type? Does it actually exist in the data from the Salesforce API?